plugins/youtube/plugin.py

```diff
@@ -88,7 +88,7 @@ def _get_video_info(self, cardinal, channel, url):
             message = self._parse_item(result['items'][0])
             cardinal.sendMsg(channel, message)
         except IndexError:
-            raise EventRejectedMessage
+            raise EventRejectedMessage, e
```

Comment:
I added it so I could see the underlying error during testing with a monkeypatched plugin runner. Feel free to remove it (though it provides useful info for debugging)